src/Raven.Server/ServerWide/RawDatabaseRecord.cs

```diff
@@ -308,8 +308,26 @@ public List<ExternalReplication> ExternalReplications
                 return _externalReplications;
             }
         }
-        
-        
+
+        public IEnumerable<PullReplicationDefinition> GetHubPullReplications()
+        {
+            if (_record.TryGet(nameof(DatabaseRecord.HubPullReplications), out BlittableJsonReaderArray bjra) == false || bjra == null) 
```

Comment:
you want to handle this exactly the same as other properties

Reply:
Sure. Done.